### ISSUE TITLE
Allow command line arguments for parameters

### ISSUE THREAD

**Issue by ppf2** [elastic/logstash#4119]
Currently, `-e` only allows you to pass in the entire block (eg. entire input block) as a parameter.  There are cases like the jdbc input plugin where it is useful to be able to pass in just the value(s) for a single parameter.  See https://github.com/logstash-plugins/logstash-input-jdbc/issues/66#issuecomment-152296564 on why `-e` is not an ideal workaround for the use case.

**Comment by ppf2** [comment]
Linking https://github.com/elastic/logstash/issues/3944 as well (likely related)

**Comment by suyograo** [comment]
For now the workaround would be to use sed, awk or other text manipulating unix tools

```
sed s/:input/\'test3\'/ sql_template.txt > input_sql.txt |./logstash -f jdbc.conf
sql_template.txt (select * from table1 where text = :input)
```

In the jdbc.conf, the statement file path will be pointing to input_sql.txt.

**Comment by fbaligand** [comment]
Hi @ppf2,

This new feature which will be released in Logstash 2.3.0 should help you :
https://github.com/elastic/logstash/blob/master/docs/static/configuration.asciidoc#using-environment-variables-in-configuration
